src/App.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
 
 import Bookings from "./Bookings";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristCards";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      {/* <header className="App-header">CYF Hotel</header> */}
+      <TouristInfoCards />
       <Bookings />
+      <Footer
+        addresses={[
```

Comment:
This works, but it'd be good to think about the naming of this prop - the things here aren't all addresses - can you think of a different name that may be a little more accurate?